ddtrace/tracer/tracer.go

```diff
@@ -57,6 +60,36 @@ type tracer struct {
 	pid string
 }
 
+type statsdClient interface {
+	Incr(name string, tags []string, rate float64) error
+	Count(name string, value int64, tags []string, rate float64) error
+	Gauge(name string, value float64, tags []string, rate float64) error
+	Timing(name string, value time.Duration, tags []string, rate float64) error
+	Close() error
+}
+
+type noopStats struct{}
+
+func (n *noopStats) Incr(name string, tags []string, rate float64) error {
```

Comment:
Just a nit, but it's better in a way for these type of methods to be written as:
```suggestion
func (*noopStats) Incr(_ string, _ []string, _ float64) error {
```
Because nothing is used.